.ci/variables/variables_py3-a2.8.yml

```diff
@@ -20,7 +20,7 @@ registry_tag: "py3-a2.8"
 
 #. registry_extra_tags:
 #+ Additional tags to put on the builded image.
-registry_extra_tags: ""
+registry_extra_tags: "v2.8 a2.8"
 
```

Comment:
registry_tag_describe_ref might be missing but will be added with https://github.com/cycloidio/docker-cycloid-toolkit/pull/22/files

Reply:
I updated the variables to match the latest dockerbuild pipeline